### PR TITLE
App customization based on the chain info and customizable theme

### PR DESCRIPTION
**Pull request by hamidra**
The changes include:
- adding a config value (config/common.json  as well as query strings) for theme which let us customize the app per this config value (currently the available values are (polkadot, kusama, kusamanft)
- reading the token symbol from the on chain configuration for the network

### CODE REVIEW

File: src/substrate-lib/SubstrateContext.js [hunk 1]
```diff
@@ -81,9 +74,33 @@ const reducer = (state, action) => {
...
+  theme = theme?.toLowerCase();
+
+  // default values
+  const giftTheme = {
+    content: `${chainInfo?.token}s`,
+    network: `${chainInfo?.chainName}`,
+  };
+  switch (theme) {
+    case 'polkadot':
+      giftTheme.content = 'DOTs';
```
Comment (erler) [inline src/substrate-lib/SubstrateContext.js:87]: Just a nit, imo "x DOT" / "x KSM" sounds more natural than "x DOTs" or "x KSMs"

**Comment by erler** [comment]
Looks good! We can then go ahead and load the sass files conditionally depending on theme I guess.

One thing I noticed when loading:
Screenshot 2021-06-18 at 10 35 47

File: src/substrate-lib/SubstrateContext.js [hunk 2]
```diff
@@ -81,9 +74,33 @@ const reducer = (state, action) => {
...
+      giftTheme.network = 'polkadot';
```
Comment (erler) [inline src/substrate-lib/SubstrateContext.js:88]: Another nit: "polkadot" is not capitalised, but "Kusama" is

**Comment by hamidra** [comment]
> Looks good! We can then go ahead and load the sass files conditionally depending on theme I guess.
> 
> One thing I noticed when loading:
> Screenshot 2021-06-18 at 10 35 47
> 
Should be fixed. It was because we were also loading the theme from chan Info. for now made it static only based on config. I'll consider to fix the UX to don't show the card till the chainInfo is loaded later.

**Comment by hamidra** [comment]
> Edit
> Followed [this tutorial](https://prawira.medium.com/react-conditional-import-conditional-css-import-110cc58e0da6) for loading scss based on themes: https://github.com/hamidra/dotdrop/tree/load-themes
> However, it takes two seconds to load any kind of styling. So there's probably a better way to do it.
Since we are deploying the two variation as separate apps, We should be good to load the theme styles statically. And it is also faster, so I don't think we need to load the scss dynamically if it makes the loading slow.

**Comment by erler** [comment]
> Since we are deploying the two variation as separate apps, We should be good to load the theme styles statically
Makes sense! lgtm

**Review by erler — APPROVED** [review]

**Pull request merged**